spec/models/event_spec.rb

```diff
@@ -59,6 +59,28 @@
   end
   
 
+  it "checks if there are unclassified applications_letters" do
+    event = FactoryGirl.create(:event)
+    accepted_application_letter = FactoryGirl.create(:application_letter_accepted, :event => event, :user => FactoryGirl.create(:user))
+    event.application_letters.push(accepted_application_letter)
+    expect(event.applications_classified?).to eq(true)
+
+    pending_application_letter = FactoryGirl.create(:application_letter, :event => event, :user => FactoryGirl.create(:user))
+    event.application_letters.push(pending_application_letter)
+    expect(event.applications_classified?).to eq(false)
+  end
+
+  it "computes the email addresses of the accepted and the rejected applications" do
+    event = FactoryGirl.create(:event)
+    accepted_application_letter_1 = FactoryGirl.create(:application_letter_accepted, :event => event, :user => FactoryGirl.create(:user))
+    accepted_application_letter_2 = FactoryGirl.create(:application_letter_accepted, :event => event, :user => FactoryGirl.create(:user))
+    accepted_application_letter_3 = FactoryGirl.create(:application_letter_accepted, :event => event, :user => FactoryGirl.create(:user))
+    rejected_application_letter = FactoryGirl.create(:application_letter_rejected, :event => event, :user => FactoryGirl.create(:user))
+    [accepted_application_letter_1, accepted_application_letter_2, accepted_application_letter_3, rejected_application_letter].each { |letter| event.application_letters.push(letter) }
```

Comment:
Can this line be replaced by
`event.application_letters += [accepted_application_letter_1, accepted_application_letter_2, accepted_application_letter_3, rejected_application_letter]`? As further improvement can the applications can be stored in an array with an descriptive name?